src/dcc.c

```diff
@@ -2414,12 +2416,16 @@ static void dcc_telnet_got_ident(int i, char *host)
   /* This is so we don't tell someone doing a portscan anything
    * about ourselves. <cybah>
    */
-  if (stealth_telnets)
+  if (stealth_telnets) {
+    /* Show here so it doesn't interfere with newline-less stealth_prompt */
+    if (allow_new_telnets)
+      dprintf(i, "(If you are new, enter 'NEW' here.)\n");
```

Comment:
Looks like a visible change to me. Do we really want to display this for case stealth telnets?

Reply:
It has been displayed before this PR so the only change is it's location. While it would be handy to be able to configure this (either by defining your own "new_telnet_prompt" or with a toggle setting), it shouldn't be part of this PR.

Reply:
Yes and No. It has been displayed before regardless of stealth_prompt. So, your patch here is an enhancement. But: Its showing the new_telnet_prompt in the wrong case. It shows it when stealth_prompt is set instead of if steal_prompt is unset. So if you would switch it to the else case, it would really be a nice enhancement :)